WooCommerce/WooCommerceTests/ViewModels/CardPresentPayments/PaymentReceiptEmailParameterDeterminerTests.swift

```diff
@@ -0,0 +1,129 @@
+import XCTest
+import TestKit
+@testable import WooCommerce
+@testable import Yosemite
+
+final class PaymentReceiptEmailParameterDeterminerTests: XCTestCase {
+    private var stores: MockStoresManager!
+
+    override func setUp() {
+        super.setUp()
+
+        stores = MockStoresManager(sessionManager: SessionManager.makeForTesting())
+        stores.whenReceivingAction(ofType: SystemStatusAction.self) { action in
+            guard case let .synchronizeSystemPlugins(_, onCompletion) = action else {
+                return
+            }
+
+            onCompletion(.success(()))
+        }
+    }
+
+    override func tearDown() {
+        stores = nil
```

Comment:
nit: maybe also call super here?
```suggestion
        stores = nil
        super.tearDown()
```

Reply:
👍